src/components/battles/animations/Ending.ts

```diff
@@ -85,6 +85,15 @@ export class Ending<TGameStartr extends FullScreenPokemon> extends GeneralCompon
             battleInfo.onComplete(outcome);
         }
 
+        if (battleInfo.endingtheme !== undefined && battleInfo.endingtheme !== "None") {
+            this.gameStarter.audioPlayer.play(battleInfo.endingtheme, {
+                alias: this.gameStarter.audio.aliases.theme,
+                loop: true,
+            });
+        } else {
+            this.gameStarter.audioPlayer.stopAll();
```

Comment:
As discussed: no need for a `stopAll()`. There could be other relevant sounds playing.